x-pack/plugins/maps/public/shared/layers/sources/kibana_tilemap_source/kibana_tilemap_source.js

```diff
@@ -51,12 +52,19 @@ export class KibanaTilemapSource extends  AbstractTMSSource {
     });
   }
 
-
   async getUrlTemplate() {
-    return this._descriptor.url;
+    const tilemap = await getKibanaTileMap();
+    if (!tilemap || !tilemap.url) {
+      throw new Error(`Unable to find map.tilemap.url configuration in the kibana.yml`);
```

Comment:
throw so we can show error message in the layer-panel when the config is invalid.